### ISSUE TITLE
Arizona Parkways, Historic and Scenic Roads

### ISSUE THREAD

**Issue by 1ec5** [ZeLonewolf/openstreetmap-americana#924]
The [Arizona Parkways, Historic and Scenic Roads](https://wiki.aaroads.com/wiki/Arizona_Parkways,_Historic_and_Scenic_Roads) program posts distinctive shields along its routes, many of which are concurrent with state routes. The design is apparently in the public domain based on its inclusion in Arizona’s MUTCD supplement. In OSM, these routes are tagged [`network=US:AZ:Scenic`](https://taginfo.openstreetmap.org/tags/network=US%3AAZ%3AScenic).

[![Route 66](https://upload.wikimedia.org/wikipedia/commons/thumb/7/74/Route66.jpg/180px-Route66.jpg)](https://commons.wikimedia.org/wiki/File:Route66.jpg) [<img src="https://upload.wikimedia.org/wikipedia/commons/f/fc/Arizona_Scenic_Road_Marker.svg" height="200" alt="Arizona scenic road marker">](https://commons.wikimedia.org/wiki/File:Arizona_Scenic_Road_Marker.svg)

**Comment by 1ec5** [comment]
The two images above differ in the color of the sky and the shapes of the mountains and cactus. Responses from [Discord](https://discord.com/channels/1025211888084193380/1120975283663687780/1152871036220620881) point out that the shield on the left is an older one, while the shield on the right is ADOT’s current official design.

**Comment by b1tw153** [comment]
I'm finally getting around to contributing the AZ Scenic shield, but I'm not so good at Git. Should I create a new branch and push shield_us_az_scenic.svg and an update to shield_defs.js?

![Arizona Parkway Construction 2](https://github.com/ZeLonewolf/openstreetmap-americana/assets/123102061/1854cc01-64fa-4510-a828-f24725d8dbaa)

**Comment by 1ec5** [comment]
Yes, exactly. You could do it entirely online from within GitHub, but building it locally allows you to see what it’ll look like. Once you push your branch, create a pull request out of it; the PR will automatically build a preview that we can all examine.

**Comment by b1tw153** [comment]
It all sounds so easy, but I have failed at every attempt to do this.

**Comment by 1ec5** [comment]
Want to pair on this over screen sharing tomorrow during the noon EST slot? We were going to have an Americana workshop then but had to cancel because the repo wasn’t quite where we needed it to be for that workshop. But helping you make this contribution would be the least we could do instead.

**Comment by b1tw153** [comment]
Thanks for the offer. Noon ET today won't work, but I'll follow up on Slack. The closest I've been able to get is creating a local branch off main, updating the two files, and staging and committing them. I'm getting 403 error on the push even though my login to Github looks fine.

I did look for the options to do this on the Github web site, but the documented methods to add a branch don't seem to be present (anymore? just in this project? because I don't have permissions?).
